components/developer/golang-115/pkg5

```diff
@@ -0,0 +1,11 @@
+{
+    "dependencies": [
+        "developer/golang-113",
+        "SUNWcs",
+        "system/library"
+    ],
+    "fmris": [
+        "developer/golang-114"
+    ],
```

Comment:
There is something wrong here!

Reply:
Jep. How does that file get generated?

Reply:
It's generated by gmake update-metadata (part of gmake publish)

Reply:
Fixed